### ISSUE TITLE
megabasterd issue

### ISSUE THREAD

**Issue by k1tsune2** [tonikelope/megabasterd#668]
for some unknown reasons my downloads are not getting past provisioning download
![Screenshot_1](https://github.com/tonikelope/megabasterd/assets/168975222/6cc7f46b-b7f2-402e-9ba4-5614c64265c7)

**Comment by ArnabIsCoding** [comment]
Same issue. Any solution to this problem? Is your issue resolved?

**Comment by k1tsune2** [comment]
> Same issue. Any solution to this problem? Is your issue resolved?

nope:/ pls do let me know if u find a solution. there seems to be no replies
